static/docs/use-cases/shared-storage-on-nfs.md

```diff
@@ -0,0 +1,148 @@
+# Shared Storage on NFS
```

Comment:
I want to rename it ... it's not about NFS only. It's about any network attached storages. We can do something like:

`Share Storage on NAS (NFS)`